reflex/event.py

```diff
@@ -509,6 +509,15 @@ def console_log(message: str | Var[str]) -> EventSpec:
     return server_side("_console", get_fn_signature(console_log), message=message)
 
 
+def back() -> EventSpec:
+    """Do a history.back on the browser.
+
+    Returns:
+        An event to go back one page.
+    """
+    return server_side("_back", get_fn_signature(back))
```

Comment:
do we need to handle this as a special case in `state.js`? seems like it would also work to have

```python
return call_script("window.history.back()")
```

Reply:
Thanks, such an improvement. But now it looks trivial :smile:

Reply:
thats good. trivial is easier to maintain 😸 

in the mid term, we're planning to refactor `state.js` to break it up into smaller parts and move more logic to the python side so there are fewer "moving parts"